cypress/support/pages/admin/pageAdministration.js

```diff
@@ -745,11 +955,12 @@ class Management {
 	}
 
 	seeEmptyExternalToolTable() {
-		cy.get(Management.#externalToolsTable)
-			.within( ()=> {
-				cy.get("tbody tr.v-data-table-rows-no-data td")
-					.should("have.text", "Keine Daten vorhanden");
-			});
+		cy.get(Management.#externalToolsTable).within(() => {
+			cy.get("tbody tr.v-data-table-rows-no-data td").should(
```

Comment:
Isn't selector should be named as variable and then use that variable here?

Reply:
Its not from this PR, its on main too. Due to prettier its showing updated.